Source/JavaScriptCore/dfg/DFGAbstractInterpreterInlines.h

```diff
@@ -1187,6 +1192,12 @@ bool AbstractInterpreter<AbstractStateType>::executeEffects(unsigned clobberLimi
         if (handleConstantDivOp(node))
             break;
 
+        if (node->mustGenerate()) {
```

Comment:
Let's check `node->isBinaryUseKind(HeapBigIntUse) || node->isBinaryUseKind(AnyBigIntUse) || node->isBinaryUseKind(BigInt32Use)` here, since otherwise, we may have non bigint in right hand side.